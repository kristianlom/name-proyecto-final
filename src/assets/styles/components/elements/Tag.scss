@import '../lib/palette';
@import '../lib/fonts';

.Tag {
  height: 32px;
  width: 38px;
  background: map-get(map-get($palette,lightTheme),background);
  border-radius: 5px;
  opacity: 1;
  font-size: 12px;
  font-family: 'Quicksand', sans-serif;
  display: flex;
  justify-content: center;
  align-items: center;
  &:hover {
    background: map-get(map-get($palette,lightTheme),white) 0% 0% no-repeat padding-box;
    box-shadow: 0px 0px 2px #00000029;
    border: 1px solid map-get(map-get($palette,primary),blue);
    border-radius: 5px;
    opacity: 1;
  }
}