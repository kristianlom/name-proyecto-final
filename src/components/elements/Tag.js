import React from 'react';
import '../../assets/styles/components/elements/Tag.scss';

const Tag = () => {
  return (
    <div className="Tag">
      tag
    </div>
  );
};

export default Tag;
