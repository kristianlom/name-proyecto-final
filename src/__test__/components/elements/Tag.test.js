import React from 'react';
import { shallow } from 'enzyme';
import Tag from '../../../components/elements/Tag';

const wrapped = shallow(<Tag />);

describe('snapshot of Tag component', () => {
  it('should render the Tag Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });
  it('renders the Titles children', () => {
    expect(wrapped.find(Tag).length).toEqual(1);
  });
});
