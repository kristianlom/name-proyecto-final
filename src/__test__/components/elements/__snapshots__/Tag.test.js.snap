// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of Tag component should render the Tag Component correctly 1`] = `ShallowWrapper {}`;
